jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/ExtendedMemoryMXBeanImpl.java

```diff
@@ -94,4 +94,12 @@ protected MemoryPoolMXBean makeMemoryPoolBean(String name, MemoryType type, int
 		return new MemoryPoolMXBeanImpl(name, type, internalID, this);
 	}
 
+	@Override
+	public String getHeapClassStatistics() {
```

Comment:
I don't think we should be added this to the MemoryMXBean.

I notice the following, not sure if this class is involved in the implementation of jcmd, but if so, OpenJ9 should implement it as well.
http://w3.hursley.ibm.com/java/docs/java8/jre/api/management/extension/com/sun/management/DiagnosticCommandMBean.html

Reply:
Moved the target code required for jcmd into java.base.